services/gateway/apis/http-version3/methods/modules/pos/posValidators.js

```diff
@@ -24,9 +24,9 @@ module.exports = {
 	rpcMethod: 'get.pos.validators',
 	tags: ['PoS'],
 	params: {
-		address: { optional: true, type: 'string', pattern: regex.ADDRESS_LISK32 },
+		address: { optional: true, type: 'string', pattern: regex.ADDRESS_LISK32_CSV, max: 419, altSwaggerKey: 'accountAddressCsv' },
 		publicKey: { optional: true, type: 'string', pattern: regex.PUBLIC_KEY },
-		name: { optional: true, type: 'string', pattern: regex.NAME },
+		name: { optional: true, type: 'string', pattern: regex.NAME_CSV, max: 209, altSwaggerKey: 'validatorNameCsv' },
```

Comment:
```suggestion
		name: { optional: true, type: 'string', pattern: regex.NAME_CSV, max: 209, altSwaggerKey: 'nameCsv' },
```

Reply:
I would prefer keeping previous name as the description is specific to validators.

```
Query validators by their registered name (can be expressed as CSV)
```

Reply:
Please ensure that adding an additional property is not causing issues with the `fastest-validator` implementation.

Reply:
Working fine with current validation.